http-internal.h

```diff
@@ -32,7 +32,7 @@ struct addrinfo;
 struct evhttp_request;
 
 /* Indicates an unknown request method. */
-#define EVHTTP_REQ_UNKNOWN_ (1<<15)
+#define EVHTTP_REQ_UNKNOWN_ (1<<31)
```

Comment:
Replaced with just 0 in 799053d, rebased patch is 847f6f944a7a1aecff8a99d85ca05d819ead1b54